sn_networking/src/driver.rs

```diff
@@ -589,7 +589,7 @@ impl NetworkBuilder {
             listen_port: self.listen_addr.map(|addr| addr.port()),
             is_client,
             is_behind_home_network: self.is_behind_home_network,
-            connected_peers: 0,
+            peers_in_rt: 0,
```

Comment:
This tracks the total number of peers in the RT rather than the "connections". So renamed it for clarity.